elpy.el

```diff
@@ -1565,8 +1591,7 @@ with a prefix argument)."
                              " --ignore="
                            " --exclude=")
                          (mapconcat #'identity
-                                    elpy-project-ignored-directories
-                                    ","))
+                                    (elpy-project-ignored-directories) ","))
```

Comment:
Could you please leave the third argument to mapconcat on a separate line? This makes the code less readable for no reason :-)